android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -196,6 +203,22 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
         }
     }
 
+    private fun setScalingMode(view: RNPlayerView, scalingMode: String) {
+        Handler(Looper.getMainLooper()).post {
+            when (scalingMode) {
+                "Zoom" -> {
+                    view.playerView?.scalingMode = ScalingMode.Zoom
+                }
+                "Stretch" -> {
+                    view.playerView?.scalingMode = ScalingMode.Stretch
+                }
+                else -> {
+                    view.playerView?.scalingMode = ScalingMode.Fit
+                }
```

Comment:
Consider using `Enum.valueOf`:
```suggestion
            view.playerView?.scalingMode = ScalingMode.valueOf(scalingMode)
```